org.gnome.Rhythmbox3.json

```diff
@@ -23,9 +23,8 @@
         "--socket=pulseaudio",
         /* Browse user's Music directory */
         "--filesystem=xdg-music",
-        /* Needed for dconf to work */
-        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
-        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        /* Migrate DConf settings from the host */
+        "--metadata=X-DConf=migrate-path=/org/gnome/rhythmbox/",
```

Comment:
This doesnt match app-id and will fail. If the app uses gsettings they will be lost.

Reply:
We'll need to add a special case in the conversion code on the host side, because it will be impossible to fix otherwise. (I don't know who or more importantly **why** somebody added "3" to the app-id in flathub either)

Reply:
It can be fixed in the sandbox with a transition period, have a pre-run script that migrates if settings are found.

Reply:
Also we do have the power to rename app-ids now. In the end (and in the start) it should be an ID upstream likes.

Reply:
The D-Bus service is called `org.gnome.Rhythmbox3`. As there never was any link or restrictions between DConf paths and app-id, I don't think we want to change either.

I'd rather get the flatpak changes merged, and require that version of Flatpak when we merge the Rhythmbox changes.

Reply:
Fair enough.